puts "What does the following error message tell you?:
  ArgumentError: wrong number of arguments (1 for 2)
  from (irb):1:in `calculate_product'
  from (irb):4
  from /Users/username/.rvm/rubies/ruby-2.5.3/bin/irb:12:in `<main>'

"

puts "We should pass two arguments inside calculate_product method,we only passed 1 instead."
